include/sonic/dom/genericnode.h

```diff
@@ -1089,6 +1117,33 @@ class GenericNode {
     return static_cast<const NodeType*>(this);
   }
   NodeType* downCast() noexcept { return static_cast<NodeType*>(this); }
+
+  template <typename StringType>
+  NodeType* atPointerImpl(const GenericJsonPointer<StringType>& pointer) const {
```

Comment:
return const

Reply:
This is private method for public interface, const is not necessary.